### PR TITLE
Added Dotcast

### PR DESCRIPTION
**Pull request by D0tcast**

### CODE REVIEW

**Comment by D0tcast** [comment]
https://kusama.subscan.io/extrinsic/17697593-2

File: registry/kusama.json [hunk 1]
```diff
@@ -166,6 +166,13 @@
         "shortDescription": "AIWeb3 Chinese community",
         "longDescription": "The mission of the AIWeb3 Chinese community is to continue building the Dotsama community, supporting Chinese content creators to focus on content creation related to Dotsama, establishing connections with different parachain projects, and providing the latest information about Dotsama. Chinese content creators can focus on researching and producing high-quality content, including dubbing, translation, and original content within Dotsama. Additionally, we will allocate more resources to support early stage content creators, helping them establish their audience and brand.",
         "isOrganization": true
-     }    
-    
+     },    
+     {
+        "address": "EAMbhBVW7SpJfH332uABbDNtsbNnaZBBXeGJwdfxkxkPm6o",
+        "name": "Dotcast",
+        "image": "https://github.com/D0tcast/opengov-delegate-registry/blob/8bc295ec75bdf4eb85ab771759fed0011fa14716/images/Dotcast%20Expect%20Chaos.png",
```
Comment (stepanLav) [inline registry/kusama.json:173]: Only images that have been posted in the current repository and comply with our guidelines are permitted, could you change that link to this one:
```suggestion
        "image": "https://raw.githubusercontent.com/nova-wallet/opengov-delegate-registry/master/images/Dotcast%20Expect%20Chaos.png",
```

**Review by stepanLav — APPROVED** [review]

**Comment by D0tcast** [comment]
Hey,

Thanks for the feedback!

we have applied the suggested change

Looking for your comments and hopping everything runs as it should be,

Thank you again!!

Regards, The Dotcast team.

**Comment by D0tcast** [comment]
Hey Stepan,

We sorry for the "TEST TUTORIAL DOTCAST" that you might see around...

It is just that, a test for a tutorial that we are producing for our
youtube channel with the aim of showing users how to do it in spanish...

Thank you and sorry to bother you or the team.

**Pull request merged**